src/bap_main.ml

```diff
@@ -195,10 +195,16 @@ let program_info =
   Term.info "bap" ~version:Config.version ~doc ~man
 let program source =
   let create
-      a b c d e f g i j k = Bap_options.Fields.create
-      a b c d e f g i j k [] in
+      passopt
+      a b c d e f g i j k = (Bap_options.Fields.create
+                               a b c d e f g i j k []), passopt in
   let open Bap_cmdline_terms in
+  let passopt : string list Term.t =
+    let doc = "Runs passes (comma separated)" in
```

Comment:
Since it is possible that this temporal hack will live for a quite a long time, a month or so, can you add a little bit better doc strings. Or, at least, we can redirect a reader to the `--<pass>` option description, by saying in `doc`, that `--passes=p1,p2,p3 is an alias to --p1 --p2 --p3. Please, use the former, as the latter is deprecated and soon will be removed"